src/main/java/lembas/core/LembasCore.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (C) 2019 Lembas Modding Team
+ * Copyright (C) 2020 Lembas Modding Team

```

Comment:
We should use a copyright range (id est `2019-2020`)

Reply:
2019 - currentYear ? I'll adjust that...

Reply:
Fixed with f7985b49c3aa999b2ce2179deb426572cc1c49d9